kubernetes/e2e_test/test_client.py

```diff
@@ -167,17 +168,64 @@ def test_portforward_raw(self):
         client = api_client.ApiClient(configuration=self.config)
         api = core_v1_api.CoreV1Api(client)
 
+        with open(os.path.join(os.path.dirname(__file__), 'port_server.py')) as fh:
+            port_server_py = fh.read()
         name = 'portforward-raw-' + short_uuid()
-        pod_manifest = manifest_with_command(
-            name,
-            ' '.join((
-                '((while true;do nc -l -p 1234 -e /bin/cat; done)&);',
-                '((while true;do nc -l -p 1235 -e /bin/cat; done)&);',
-                'sleep 60',
-            ))
```

Comment:
what about using the k8s test image below, instead of writing your own echo server?
https://github.com/kubernetes/kubernetes/tree/master/test/images/agnhost#netexec

Reply:
This is a test of raw socket usage, not http based. Granted, the code could send the needed http request using the raw socket. It then has to pull in what is being returned and compare against a hard coded assumption of the return value. If the agnhost image ever changes the return value, the test will fail.

This is also a test of raw socket usage against multiple ports. I do not know if agnhost can orchestrate that. I am not a go guru and would have to spend time understanding exactly what it is this agnhost container is doing.

My preference is to control exactly what is occurring on the test container and not create external dependencies. This is a python client library, and I find it simpler to understand a python based container, rather then a go based implemention.

Reply:
sounds good. thanks for the fix